libraries/dynamic-ui-react/src/style/abstracts/_utilities.scss

```
@@ -55,6 +55,20 @@ $utilities: map-merge(
         )
       )
     ),
+    "font-weight": (
+      property: font-weight,
+      responsive: true,
+      class: fw,
+      values: (
+        lighter: $font-weight-lighter,
+        light: $font-weight-light,
+        normal: $font-weight-normal,
+        medium: $font-weight-medium,
+        semibold: $font-weight-semibold,
+        bold: $font-weight-bold,
+        bolder: $font-weight-bolder
+      )
+    ),
```

sería algo así, pero prueba la sugerencia y no la apliques de una :) , porque es complicado escribir esto acá 

```suggestion
    "font-weight": (
      map-get($utilities, "font-weight"),
      (
        responsive: true
      )
    ),
```